src/setup-wp-env/create-wp-env-config.ts

```diff
@@ -0,0 +1,91 @@
+import { minimist, fs } from 'zx';
+
+async function main() {
+	const {
+		wp,
+		php,
+		plugins,
+		themes,
+		mappings,
+		dir,
+		'active-theme': activeTheme,
+	} = getOptions({
+		wp: { type: 'string', default: null },
+		php: { type: 'string', default: null },
+		plugins: { type: 'string', default: '' },
+		themes: { type: 'string', default: '' },
+		'active-theme': { type: 'string', default: '' },
+		mappings: { type: 'string', default: '' },
+		dir: { type: 'string', default: './' },
+	});
+
+	const config = {
+		core: wp ? `WordPress/Wordpress#${wp}` : null,
+		phpVersion: php ? php : null,
+		themes: normalizeArray(themes),
+		mappings: normalizeMappings(mappings),
+		plugins: normalizeArray(plugins),
+		lifecycleScripts: {
+			afterStart: prepareCommands(
+				['cli', 'tests-cli'],
+				[
+					activeTheme && `wp theme activate ${activeTheme}`,
+					`wp rewrite structure '/%postname%' --hard`,
+				],
+			),
+		},
+	};
+
+	await fs.ensureDir(dir);
+	await fs.writeJSON(`${dir}/.wp-env.json`, config, { spaces: 2 });
+}
+
+function getOptions(
+	args: Record<
+		string,
+		{ type: 'string' | 'boolean'; default: string | boolean | null }
+	>,
+) {
+	const entries = Object.entries(args);
+
+	return minimist(process.argv.slice(2), {
+		string: entries
+			.filter(([, { type }]) => type === 'string')
+			.map(([key]) => key),
+		boolean: entries
+			.filter(([, { type }]) => type === 'boolean')
+			.map(([key]) => key),
+		default: Object.fromEntries(
+			entries.map(([key, { default: value }]) => [key, value]),
+		),
+	});
+}
+
+function normalizeMappings(mappings: string) {
```

Comment:
```suggestion
function mappingsFromString(mappings: string) {
```

maybe?
and maybe do something similar for the array I said before?

Reply:
Done